twpca/model.py

```diff
@@ -315,37 +320,18 @@ def fit(self, optimizer=None, niter=1000, lr=1e-3, progressbar=True):
             lr (optional): float, learning rate for the optimizer (default: 1e-3)
             progressbar (optional): whether to print a progressbar (default: True)
         """
-
```

Comment:
Just wanted to check, now the syntax for training with different learning rates would be to call `fit` multiple times, but make sure `optimizer` is None the second time?